os/se/ameba/rtl_security_api_wrapper.c

```diff
@@ -283,13 +312,70 @@ uint32_t rtl_read_factory_key_wrapper(hal_data *data)
 	}
 
 	if (buf_flash != NULL) {
-		free(buf_flash);
+		kmm_free(buf_flash);
 	}
 	if (buf_plain != NULL) {
-		free(buf_plain);
+		kmm_free(buf_plain);
+	}
+	if (buf_crypt != NULL) {
+		kmm_free(buf_crypt);
+	}
+
+	return ret;
+}
+
+uint32_t rtl_write_factory_key_wrapper(hal_data *data)
```

Comment:
There is no code to call these new functions, rtl_write_factory_key_wrapper and rtl_write_factory_cert_wrapper. Could you let me know the purpose (or plan) of them?

Reply:
rtl_write_factory_cert_wrapper() and rtl_write_factory_key_wrapper() is for future reference. When want to write factory key or cert can directly use these API.